modules/dnn/test/test_precomp.hpp

```diff
@@ -47,6 +47,6 @@
 #include "opencv2/core/ocl.hpp"
 
 #include "opencv2/dnn.hpp"
-#include "test_common.hpp"
+#include "../test/test_common.hpp"
```

Comment:
Does this change required?

Reply:
I have
```
fatal error: test_common.hpp: No such file or directory
```
with the following configuration:

```
cmake -DCMAKE_INSTALL_PREFIX=/usr/local \
  -DCMAKE_BUILD_TYPE=Release \
  -DWITH_OPENCL=ON \
  -DWITH_LAPACK=OFF \
  -DWITH_JASPER=OFF \
  -DBUILD_DOCS=OFF \
  -DBUILD_TESTS=ON \
  -DWITH_OPENMP=OFF \
  -DBUILD_PERF_TESTS=ON \
  -DWITH_OPENEXR=OFF \
  -DWITH_WEBP=OFF \
  -DWITH_TIFF=OFF \
  -DWITH_INF_ENGINE=ON \
  -DWITH_VULKAN=OFF \
  -DENABLE_CXX11=ON \
  -DBUILD_EXAMPLES=OFF \
  -DINF_ENGINE_RELEASE=2019010000 \
  -DBUILD_LIST=ts,dnn,python2 ..
```

note: without IE it's ok.

Reply:
removed it